tools/pharmcat/pharmcat.xml

```diff
@@ -0,0 +1,215 @@
+<tool id="pharmcat" name="pharmcat" version="@WRAPPER_VERSION@+@VERSION_SUFFIX@" profile="20.01">
+    <macros>
+        <token name="@WRAPPER_VERSION@">1.3.1</token>
+        <token name="@VERSION_SUFFIX@">galaxy0</token>
+    </macros>
+    <requirements>
+        <container type="docker">pgkb/pharmcat:1.3.1</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    ## NAMECALLER FUNCTION
+        #if $function_select.function == 'name':
+            ln -s '$function_select.input' ./input.vcf &&
+            java -cp /pharmcat/pharmcat.jar
+            org.pharmgkb.pharmcat.haplotype.NamedAlleleMatcher
+            -vcf input.vcf
+            -json output.json
+            -html output.html
+
+    ## PHENOTYPE CALLER FUNCTION
+        #else if $function_select.function == 'pheno':
+            #if $function_select.method.pheno_function == 'vcf':
+                ln -s '$function_select.method.input' ./input.vcf &&
+                java -cp /pharmcat/pharmcat.jar
+                org.pharmgkb.pharmcat.phenotype.Phenotyper
+                -vcf input.vcf
+            #else:
+                ln -s '$function_select.method.input' ./input.json &&
```

Comment:
those ln can be moved out of the if/else

Reply:
Each of those have a different file type and require their extensions be correct. Since the input is inconsistent, it needs to be in there

Reply:
ok